src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/PrisonerSearchResourceIntTest.kt

```diff
@@ -126,8 +131,8 @@ class PrisonerSearchResourceIntTest : ResourceTest() {
               assertThat(nonDtoReleaseDateType).isEqualTo(NonDtoReleaseDateType.ARD)
               assertThat(confirmedReleaseDate).isEqualTo("2018-04-19") // TODO test where OFFENDER_RELEASE_DETAILS.RELEASE_DATE is null, this should be null and NOT default to AUTO_RELEASE_DATE
               assertThat(releaseDate).isEqualTo("2018-04-19")
-              assertThat(additionalDaysAwarded).isEqualTo(0)
             }
+            assertThat(sentenceDetail?.additionalDaysAwarded).isNull()
```

Comment:
Bizarrely `assertThat(sentenceDetail!!.additionalDaysAwarded).isNull()` throws a null pointer exception so I had to move this out of the `with` above.